mmv1/third_party/terraform/website/docs/r/container_cluster.html.markdown

```diff
@@ -745,6 +745,13 @@ ephemeral_storage_config {
   local_ssd_count = 2
 }
 ```
+* `ephemeral_storage_local_ssd_config` - (Optional) Parameters for the ephemeral storage filesystem. If unspecified, ephemeral storage is backed by the boot disk. Structure is [documented below](#nested_ephemeral_storage_local_ssd_config).
```

Comment:
nit: Other than the "Structure is documented below." bit here, do you mind copying the descriptions for these fields into the `Description` value on the resource exactly? That's used by Config Connector to generate docs.